modeling-cmds/src/def_enum.rs

```diff
@@ -542,6 +542,13 @@ define_modeling_cmd_enum! {
             pub uv: Point2d<f64>,
         }
 
+        ///Obtains the surface "center of mass"
+        #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, ExecutionPlanFromMemory, ModelingCmdVariant)]
+        pub struct FaceGetCenter {
+            /// Which face is being queried.
+            pub object_id: Uuid,
```

Comment:
Shouldn't this be "face_id" as you're not querying an overall object, you're querying one specific face of an object?

Reply:
opted for consistency, see FaceGetGradient + FaceGetPosition which already use object_id